src/apps/backend/modules/authentication/internals/otp/otp_util.py

```diff
@@ -33,3 +18,15 @@ def convert_otp_bson_to_otp(otp_bson: dict[str, Any]) -> OTP:
             phone_number=validated_otp_data.phone_number,
             status=validated_otp_data.status,
         )
+
+    @staticmethod
```

Comment:
Any reason why change the order of methods? If there is no business logic change, revert the changes in this file.